docs/compression_algorithms/post_training/Quantization.md

```diff
@@ -91,3 +91,22 @@ NNCF provides the examples of Post-Training Quantization where you can find the
 function: [PyTorch](../../../examples/post_training_quantization/torch/mobilenet_v2/README.md), [TensorFlow](../../../examples/post_training_quantization/tensorflow/mobilenet_v2/README.md), [ONNX](../../../examples/post_training_quantization/onnx/mobilenet_v2/README.md), and [OpenVINO](../../../examples/post_training_quantization/openvino/mobilenet_v2/README.md)
 
 In case the Post-Training Quantization algorithm could not reach quality requirements you can fine-tune a quantized pytorch model. Example of the Quantization-Aware training pipeline for a pytorch model could be found [here](../../../examples/quantization_aware_training/torch/resnet18/README.md).
+
+## Dataloader Usage
+
+```batch_size``` is a parameter of a dataloader that refers to the number of samples or data points propagated through the neural network in a single pass.
+
+When utilizing post-training quantization with NNCF, the recommendation is to use a ```batch_size``` of 1 for your dataloader.
+
+However, NNCF supports dataloaders with arbitrary batch sizes, but there are some limitations that need to be taken into consideration. For some architectures, such as transformer-like models, statistics collection for ```batch_size``` greater than 1 could lead to inaccurate results.
+
+If the results after quantization with a batch size greater than 1 are inaccurate, the following recommendations apply:
```

Comment:
What kind of model does NNCF support quantization with calibration dataset with batch_size > 1.

Reply:
Add description of unsupported models